src/GraphQL/DocumentWriter.cs

```diff
@@ -0,0 +1,32 @@
+using System.IO;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace GraphQL
+{
+    public interface IDocumentWriter
+    {
+        Task WriteAsync<T>(Stream stream, T value);
+    }
+
+    public static class DocumentWriterExtensions
+    {
+        internal static readonly Encoding Utf8Encoding = new UTF8Encoding(false);
```

Comment:
`Utf8Encoding` can be `private` now after separation.

Reply:
Should be private now